### PR TITLE
ExportDefaultSpecifier/ExportNamespaceSpecifier - export extensions

### PR DESCRIPTION
**Pull request by hzoo**
https://github.com/leebyron/ecmascript-more-export-from

[babel visitors](https://github.com/babel/babel/blob/745f9fb5bd9dd5600ae0264d720ef0e16cc66228/packages/babel-types/src/definitions/experimental.js#L59-L77)

```
define("ExportDefaultSpecifier", {
  visitor: ["exported"],
  aliases: ["ModuleSpecifier"]
});

define("ExportNamespaceSpecifier", {
  visitor: ["exported"],
  aliases: ["ModuleSpecifier"]
});
```

Should probably look at https://github.com/leebyron/ecmascript-more-export-from/blob/master/ESTree.md again

### CODE REVIEW

File: test/lib/elements/types/ExportNamedDeclaration.js [hunk 1]
```diff
@@ -75,6 +75,26 @@ describe('ExportNamedDeclaration', () => {
         expect(statement.source.value).to.equal('m');
     });
 
+    it('should accept a ExportDefaultSpecifier from another module - exportExtensions', () => {
+        var statement = parseAndGetStatement('export x from "m";');
```
Comment (mdevils) [inline test/lib/elements/types/ExportNamedDeclaration.js]: I think it's also good to have test like this: `export {x as y, z as w} from "m"` and `export {default as i} from "m"`

**Comment by hzoo** [comment]
Oh I guess we will wait for 6.0 for this one. Also revert removing `async ()` then

**Comment by mdevils** [comment]
Great job!

**Comment by mdevils** [comment]
Will you revert removing `async`?

**Pull request merged**